ui.frontend/src/index.js

```diff
@@ -38,4 +38,4 @@ const Actions = {
     Click, Change, Submit, Blur, AddItem, RemoveItem
 }
 
```

Comment:
we should create a wrapper for File Attachment related APIs. it becomes easy for developer to understand and use.
```
const FileAttachmentUtils = {
 FileObject, extractFileInfo
}
export {...., FileAttachmentUtils}
```